internal/cmd/list.go

```diff
@@ -107,6 +92,19 @@ func list(cli *CLI) error {
 	return writeKubeconfig(user, destinations, grants)
 }
 
+func destinationForResourceExists(resource string, destinations []api.Destination) bool {
+	for _, d := range destinations {
+		if isResourceForDestination(resource, d.Name) {
+			return true
+		}
+	}
+	return false
+}
+
+func isResourceForDestination(resource string, destination string) bool {
```

Comment:
If we change our `api.Grant` type to have a `Destination` field, then I think we can remove this function and compare the names directly.  

I think we need to make a similar change to `ListGrants` to allow querying for all the grants of a destination at once (instead of querying for each namespace separately).